Src/Support/GoogleApis.Core/Apis/Http/ConfigurableMessageHandler.cs

```diff
@@ -173,6 +174,19 @@ public void RemoveExecuteInterceptor(IHttpExecuteInterceptor interceptor)
 
         #endregion
 
+        private int _loggingRequestId = 0;
+
+        private ILogger _instanceLogger = Logger;
+
+        /// <summary>
+        /// For testing only

```

Comment:
I would add to the comment here to indicate the purpose of it - it took me a while to realize that it would default to the static Logger property, but would allow more fine-grained testing.